ci/install-pktgen/helper-functions.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# simple print function that takes a single argument
+# prints the argument as a header wrapped in dashes and pipes
+print_header() {
```

Comment:
don't we already have this in the general helper-functions.h? Any reason to maintain multiple ones over just maintain a singular shared one? One shared helper-functions file makes more sense to me

Reply:
I do need different helper functions because for example, CI uses one for `manager.sh`, workers source it in `worker.sh` and Pktgen setup requires a different one. I will trim them because I found some redundancies